src/app/components/cds/inputs/ListSort.js

```diff
@@ -1,13 +1,71 @@
 // DESIGNS: https://www.figma.com/file/7ZlvdotCAzeIQcbIKxOB65/Components?type=design&node-id=1475-46077&mode=design&t=G3fBIdgR6AWtOlNu-4
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, defineMessages } from 'react-intl';
 import ArrowDropUpIcon from '../../../icons/arrow_drop_up.svg';
 import ArrowDropDownIcon from '../../../icons/arrow_drop_down.svg';
 import Tooltip from '../alerts-and-prompts/Tooltip';
 import Select from './Select';
 import styles from './ListSort.module.css';
 
+const sortLabels = defineMessages({
+  sortTitle: {
+    id: 'searchResults.sortTitle',
```

Comment:
For all strings under this `defineMessages` call, not only this: why the `id` is `searchResults` since the component name is `ListSort`?

Reply:
I'm just trying to avoid a few retranslations while keeping the set self-consistent. I'm hoping we'll eventually drop component prefixes some point in a not so distant future.

Reply:
Gotcha!